### PR TITLE
Added test with special chars in group

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
Previously failing situation returning 101.

### CODE REVIEW

**Comment by MorrisJobke** [comment]
You should remove the group afterwards:

```
  Scenario: Getting all groups            # /ssd/jenkins/workspace/ocs-api-integration-tests-ci/build/integration/features/provisioning-v1.feature:141
    Given As an "admin"                   # FeatureContext::asAn()
[Mon Jan 11 05:11:22 2016] ::1:48062 [200]: /ocs/v2.php/cloud/groups/new-group
[Mon Jan 11 05:11:22 2016] ::1:48063 [200]: /ocs/v2.php/cloud/groups/new-group
    And group "new-group" exists          # FeatureContext::assureGroupExists()
[Mon Jan 11 05:11:22 2016] ::1:48064 [200]: /ocs/v2.php/cloud/groups/admin
[Mon Jan 11 05:11:22 2016] ::1:48065 [200]: /ocs/v2.php/cloud/groups/admin
    And group "admin" exists              # FeatureContext::assureGroupExists()
[Mon Jan 11 05:11:22 2016] ::1:48066 [200]: /ocs/v1.php/cloud/groups
    When sending "GET" to "/cloud/groups" # FeatureContext::sendingTo()
    Then groups returned are              # FeatureContext::theGroupsShouldBe()
      | admin     |
      | new-group |
      Failed asserting that two arrays are equal.
      --- Expected
      +++ Actual
      @@ @@
       Array (
      -    0 => 'admin'
      -    1 => 'new-group'
      +    0 => 'España'
      +    1 => 'admin'
      +    2 => 'new-group'
       )
```

File: build/integration/features/provisioning-v1.feature [hunk 1]
```diff
@@ -134,6 +144,7 @@ Feature: provisioning
 		And group "admin" exists
 		When sending "GET" to "/cloud/groups"
 		Then groups returned are
+		    | España |
```
Comment (rullzer) [inline build/integration/features/provisioning-v1.feature]: Indentation is off

**Comment by rullzer** [comment]
I rebased and squased the last commit.
Also I changed the author of the first commit to your name+email @SergioBertolinSG instead of root ;)

:+1:

**Comment by SergioBertolinSG** [comment]
thanks @rullzer

**Comment by MorrisJobke** [comment]
:+1:

**Pull request merged**